types/http-assert/index.d.ts

```diff
@@ -10,6 +10,7 @@
  * @param opts custom properties to attach to the error object
  */
 declare function assert(value: any, status?: number, msg?: string, opts?: {}): void;
+declare function assert(value: any, status?: number, opts?: {}): void;
```

Comment:
Consider sending a follow-up PR to make both of these `object` instead of `{}`. `{}` is likely too permissive.

Reply:
ok, thanks